MAINTAINERS.yml

```diff
@@ -231,15 +231,31 @@ Build system:
     labels:
         - "area: Build System"
 
-C library:
+"C++":
     status: orphaned
+    collaborators:
```

Comment:
@stephanosio  did you check with the folks you have added here?

Reply:
Removed @pabigot -- he is no longer involved in this area.

Reply:
Sorry but I currently have no spare time left to take this up. So feel free to remove me.

Reply:
@vanwinkeljan Thanks for confirming -- removed.